tests/routes/v1/test_cptd_endpoints.py

```diff
@@ -30,13 +33,14 @@ def logger_messages():
 
 @pytest.fixture
 def process_mock():
-    with patch("crawler.blueprints.cherrypicker_test_data.process") as process:
+    with patch("crawler.routes.common.cherrypicker_test_data.process") as process:
         yield process
 
 
 @pytest.mark.parametrize("json", [{}, {"run_id": None}])
-def test_generate_endpoint_invalid_json(json, client, logger_messages):
-    response = client.post("/cherrypick-test-data", json=json)
+@pytest.mark.parametrize("endpoint_path", ENDPOINT_PATHS)
```

Comment:
Does this post to both root and v1 endpoints?

Reply:
Yes they both get tested.  Of course they should be identical, but I will be removing the root endpoint as soon as I've changed Lighthouse and I'll strip this back to just the v1 endpoint in the test.